include/vcpkg/base/json.h

```diff
@@ -24,7 +24,12 @@ namespace vcpkg::Json
         {
             Lf,
             CrLf
-        } newline_kind = Newline::Lf;
+        } newline_kind =
+#ifdef _WIN32
+            Newline::CrLf;
```

Comment:
I'm extremely opposed to changing the default to CRLF. Having `format-manifest` preserve broken if it is already there is OK but introducing platform dependence into everything vcpkg does like this is unacceptable IMO.

Reply:
Fair. TBH, There's the possibility of starting tugs of war either way but at least respecting the preexisting newlines when modifying a file will help end them early.

I'll add what you say as a comment.

This change is also probably the reason some of the PR validation tests failed on Windows. One of them, I see it comparing files in Powershell with "-ne", which will trip newline differences. I... tried to run all tests as well as I could but must have missed the e2e somehow (pointers appreciated).